pkg/adaptor/mongodb/reader.go

```diff
@@ -140,7 +140,7 @@ func (r *Reader) iterateCollection(mgoSession *mgo.Session, in <-chan string, do
 func (r *Reader) catQuery(c string, lastID interface{}, mgoSession *mgo.Session) *mgo.Query {
 	query := bson.M{}
 	if lastID != nil {
-		query = bson.M{"_id": bson.M{"$gte": lastID}}
+		query = bson.M{"_id": bson.M{"$gt": lastID}}
```

Comment:
switched this to `$gt` because mgo caches the iterator and was causing duplicate documents to be read during server restarts